### PR TITLE
Documentation for Oracle Debezium connector

### PR DESCRIPTION
**Pull request by dlg99**
### Motivation

Add documentation for new connector.

### Modifications

Added documentation.

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

This change is a documentation only.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

No

### Documentation

Check the box below and label this PR (if you have committer privilege).

Need to update docs? 

- [X] doc

### CODE REVIEW

**Review by michaeljmarshall — APPROVED** [review]

**Review by eolivelli — APPROVED** [review]
Great!

@Anonymitaet PTAL

File: site2/docs/io-debezium-source.md [hunk 1]
```diff
@@ -458,6 +458,58 @@ This example shows how to change the data of a MongoDB table using the Pulsar De
...
     ```
    
+## Example of Oracle
+
+### Packaging
+
+Oracle connector does not include Oracle JDBC driver and the end user will need to package it with the connector.
```
Comment (Anonymitaet) [inline site2/docs/io-debezium-source.md:465]:
```suggestion
Oracle connector does not include Oracle JDBC driver and you need to package it with the connector.
```

File: site2/docs/io-debezium-source.md [hunk 2]
```diff
@@ -458,6 +458,58 @@ This example shows how to change the data of a MongoDB table using the Pulsar De
...
+Major reasons for not including the drivers are the variety of versions and Oracle licensing. It is recommended using the driver provided with your Oracle DB installation, or you can [download](https://www.oracle.com/database/technologies/appdev/jdbc.html) one.
```
Comment (Anonymitaet) [inline site2/docs/io-debezium-source.md:466]:
```suggestion
Major reasons for not including the drivers are the variety of versions and Oracle licensing. It is recommended to use the driver provided with your Oracle DB installation, or you can [download](https://www.oracle.com/database/technologies/appdev/jdbc.html) one.
```

File: site2/docs/io-debezium-source.md [hunk 3]
```diff
@@ -458,6 +458,58 @@ This example shows how to change the data of a MongoDB table using the Pulsar De
...
+Integration test have an [example](https://github.com/apache/pulsar/blob/e2bc52d40450fa00af258c4432a5b71d50a5c6e0/tests/docker-images/latest-version-image/Dockerfile#L110-L122) of packaging the driver into the connector nar file. 
+
+### Configuration
+
+Debezium [requires](https://debezium.io/documentation/reference/1.5/connectors/oracle.html#oracle-overview) Oracle DB with LogMiner or XStream API enabled. 
+Supported options and steps for enabling them vary from version to version of Oracle DB.
+Steps outlined in the [documentation](https://debezium.io/documentation/reference/1.5/connectors/oracle.html#oracle-overview) and used in the [integration test](https://github.com/apache/pulsar/blob/master/tests/integration/src/test/java/org/apache/pulsar/tests/integration/io/sources/debezium/DebeziumOracleDbSourceTester.java) may or may not work for the version and edition of Oracle DB you are using.
+Please refer to the documentation for Oracle DB as needed.
```
Comment (Anonymitaet) [inline site2/docs/io-debezium-source.md:474]: Can you add a reference link here?

File: site2/docs/io-debezium-source.md [hunk 4]
```diff
@@ -458,6 +458,58 @@ This example shows how to change the data of a MongoDB table using the Pulsar De
...
+
+Similarly to other connectors, you can use JSON or YAMl to configure the connector.
```
Comment (Anonymitaet) [inline site2/docs/io-debezium-source.md:476]: Like other connector docs, can you give a JSON example?

File: site2/docs/io-debezium-source.md [hunk 5]
```diff
@@ -458,6 +458,58 @@ This example shows how to change the data of a MongoDB table using the Pulsar De
...
+    key.converter: "org.apache.kafka.connect.json.JsonConverter"
+    typeClassName: "org.apache.pulsar.common.schema.KeyValue"
+    database.history: "org.apache.pulsar.io.debezium.PulsarDatabaseHistory"
+    database.tcpKeepAlive: "true"
+    decimal.handling.mode: "double"
+    database.history.pulsar.topic: "debezium-oracle-source-history-topic"
+    database.history.pulsar.service.url: "pulsar://127.0.0.1:6650"
+```
+
+Full list of configuration properties supported by Debezium is available in the [documentation](https://debezium.io/documentation/reference/1.5/connectors/oracle.html#oracle-connector-properties).
```
Comment (Anonymitaet) [inline site2/docs/io-debezium-source.md:510]:
```suggestion
For the full list of configuration properties supported by Debezium, see [Debezium Connector for Oracle](https://debezium.io/documentation/reference/1.5/connectors/oracle.html#oracle-connector-properties).
```

File: site2/docs/io-debezium-source.md (hunk 3 above)
Comment (dlg99) [reply site2/docs/io-debezium-source.md]: done

File: site2/docs/io-debezium-source.md (hunk 4 above)
Comment (dlg99) [reply site2/docs/io-debezium-source.md:476]: done

**Pull request merged**